plugins/outputs/kafka/kafka.go

```diff
@@ -292,6 +301,16 @@ func (k *Kafka) Init() error {
 		k.TLSKey = k.Key
 	}
 
+	if k.Socks5ProxyEnabled {
+		config.Net.Proxy.Enable = true
+
+		dialer, err := k.Socks5ProxyConfig.GetDialer()
+		if err != nil {
+			return fmt.Errorf("Error while connecting to proxy server: %s", err)
```

Comment:
In golang error messages should start with a lowercase letter.
```suggestion
			return fmt.Errorf("connecting to proxy server failed: %s", err)
```

Reply:
Looks good, i commited it straight from github.